ext/gd/tests/bug75437.phpt

```diff
@@ -13,5 +13,5 @@ var_dump($rf->getNumberOfParameters());
 var_dump($rf->getNumberOfRequiredParameters());
 ?>
 --EXPECT--
-int(3)
+int(4)
 int(1)
```

Comment:
This looks like a unnecessary test.

Reply:
Yeah, I agree. I can remove it when I commit again